### PR TITLE
Added Spin CMS to tha apps list

### PR DESCRIPTION
**Pull request by alvinbakker**
New CMS added to the list

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -7278,6 +7278,17 @@
 			"implies": "Python",
 			"website": "http://sphinx.pocoo.org"
 		},
+		"SpinCMS": {
+			"cats": [
+				"1"
+			],
+			"headers": {
+				"Set-Cookie": "spincms_session"
```
Comment (jvoisin) [inline src/apps.json]: Please prefix the regexp with `^` and suffix it with `$` if possible.

File: src/apps.json [hunk 2]
```diff
@@ -7278,6 +7278,17 @@
...
+			},
+			"icon": "SpinCMS.png",
+			"implies": "PHP",
+			"website": "http://www.spin.cw"
```
Comment (jvoisin) [inline src/apps.json]: I fail to see where the CMS is described on your website. It only says that you're using it, but that's it.

File: src/apps.json (hunk 1 above)
Comment (alvinbakker) [inline src/apps.json]: It works the same as with Laravel, so the cookie is always named this way

File: src/apps.json (hunk 2 above)
Comment (alvinbakker) [inline src/apps.json]: We are currently working on our new website which will include a better description of the CMS

Comment (alvinbakker) [inline src/apps.json]: But this might take some time though. So hope it can be added for now

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**